### PR TITLE
Release v1.33.0

### PR DESCRIPTION
**Pull request by ElijahQuinones**

### CODE REVIEW

File: charts/aws-ebs-csi-driver/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
 # Helm chart
+## v2.33.0
+* Bump driver version to `v1.32.0`
+* Bump CSI sidecar container versions
+* Add fix for enableLinux node parameter ([#2078](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2078), [@ElijahQuinones](https://github.com/ElijahQuinones))
+* fix dnsConfig indentation in controller template file ([#2084](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2084), [@cHiv0rz](https://github.com/cHiv0rz))
```
Comment (ConnorJC3) [inline charts/aws-ebs-csi-driver/CHANGELOG.md:6]: np: Capitalize `fix` (all changelog entries should be capitalized)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
+# v1.33.0
```
Comment (AndrewSirenko) [inline CHANGELOG.md:1]: I think we should add the announcement from line 19 in v1.32.0 here too. But change "next minor version" to "this version"

Thoughts @ConnorJC3 ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+
+### Bug Fixes
+* Bump GCR sidecars that reference broken tags ([#2091](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2091), [@ConnorJC3](https://github.com/ConnorJC3))
+* govulncheck: Bump google.golang.org/grpc to v1.64.1 ([#2085](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2085), [@ConnorJC3](https://github.com/ConnorJC3))
+* Change coalescer InputType from comparable to any ([#2083](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2083), [@ConnorJC3](https://github.com/ConnorJC3))
+* Bump go version to fix govulncheck failure ([#2080](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2080), [@ConnorJC3](https://github.com/ConnorJC3))
+* Use new client token when CreateVolume returns IdempotentParameterMismatch ([#2075](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2075), [@ConnorJC3](https://github.com/ConnorJC3))
+
+### Improvements
+* fix function name in comment #2088 ([#2088](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2088), [@augustkang](https://github.com/augustkang))
```
Comment (ConnorJC3) [inline CHANGELOG.md:15]: np: Capitalize fix (or even consider dropping this PR from the list as it's not a user-facing change)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,20 @@
+# v1.33.0
+### Notable Changes
+* Migrate CreateVolume response topology to standard label topology.kubernetes.io/zone ([#2086](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2086), [@ConnorJC3](https://github.com/ConnorJC3))
+* Modifying Tags On Existing Volume ([#2082](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2082), [@mdzraf](https://github.com/mdzraf))
```
Comment (AndrewSirenko) [inline CHANGELOG.md:4]: Suggestion: Rename to "Add ability to modify EBS volume tags via VolumeAttributesClass"

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,20 @@
...
+* Add --kubeconfig flag for out-of-cluster auth ([#2081](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2081), [@cartermckinnon](https://github.com/cartermckinnon))
+
+### Bug Fixes
+* Bump GCR sidecars that reference broken tags ([#2091](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2091), [@ConnorJC3](https://github.com/ConnorJC3))
+* govulncheck: Bump google.golang.org/grpc to v1.64.1 ([#2085](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2085), [@ConnorJC3](https://github.com/ConnorJC3))
+* Change coalescer InputType from comparable to any ([#2083](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2083), [@ConnorJC3](https://github.com/ConnorJC3))
```
Comment (AndrewSirenko) [inline CHANGELOG.md:10]: Internal facing, please move to improvements or delete entirely.

File: CHANGELOG.md (hunk 3 above)
Comment (AndrewSirenko) [inline CHANGELOG.md:15]: Please capitalize this F. It's okay for Changelog sentence to be different from PR title. (Usually we pick a more customer friendly summary for changelog)

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,20 @@
...
+* Bump go version to fix govulncheck failure ([#2080](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2080), [@ConnorJC3](https://github.com/ConnorJC3))
+* Use new client token when CreateVolume returns IdempotentParameterMismatch ([#2075](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2075), [@ConnorJC3](https://github.com/ConnorJC3))
+
+### Improvements
```
Comment (AndrewSirenko) [inline CHANGELOG.md:14]: Add dependency upgrade PR here. 

`Bump dependencies for driver release v1.33.0 ([#2094](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2094), [@ElijahQuinones](https://github.com/ElijahQuinones))`

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,20 @@
+# v1.33.0
+### Notable Changes
+* Migrate CreateVolume response topology to standard label topology.kubernetes.io/zone ([#2086](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2086), [@ConnorJC3](https://github.com/ConnorJC3))
+* Modifying Tags On Existing Volume ([#2082](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2082), [@mdzraf](https://github.com/mdzraf))
+* Add --kubeconfig flag for out-of-cluster auth ([#2081](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2081), [@cartermckinnon](https://github.com/cartermckinnon))
+
+### Bug Fixes
+* Bump GCR sidecars that reference broken tags ([#2091](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2091), [@ConnorJC3](https://github.com/ConnorJC3))
+* govulncheck: Bump google.golang.org/grpc to v1.64.1 ([#2085](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2085), [@ConnorJC3](https://github.com/ConnorJC3))
```
Comment (AndrewSirenko) [inline CHANGELOG.md:9]: Remove the govulncheck component? Maybe add "To address CVE-[CVE Number]

File: CHANGELOG.md (hunk 2 above)
Comment (AndrewSirenko) [reply CHANGELOG.md:1]: Jinx! See Connor's comment above. Resolving duplicate...

File: charts/aws-ebs-csi-driver/CHANGELOG.md [hunk 8]
```diff
@@ -1,4 +1,10 @@
 # Helm chart
+## v2.33.0
+* Bump driver version to `v1.32.0`
```
Comment (torredil) [inline charts/aws-ebs-csi-driver/CHANGELOG.md:3]: Bump driver verstion to `v1.33.0`.

**Review by ConnorJC3 — COMMENTED** [review]
/lgtm

**Review by torredil — APPROVED** [review]
/ok-to-test
/label tide/merge-method-squash
/lgtm

**Pull request merged**